go.mod

```diff
@@ -5,5 +5,6 @@ go 1.13
 require (
 	github.com/hashicorp/consul/api v1.4.0
 	github.com/lib/pq v1.3.0
+	github.com/segmentio/kafka-go v0.3.5
```

Comment:
I can't find usage of `github.com/segmentio/kafka-go`
Is it used?

Reply:
fixed